clarity.d.ts

```diff
@@ -117,13 +117,15 @@ interface IShadowDomNode extends HTMLDivElement {
   ignore: boolean;  /* Flag to avoid sending data for that node */
 }
 
-interface ILayoutRectangle {
+interface IBoxModel {
   x: number; /* X coordinate of the element */
   y: number; /* Y coordinate of the element */
   width: number; /* Width of the element */
   height: number; /* Height of the element */
+  color: string; /* Color for visual aid */
```

Comment:
>color [](start = 2, length = 5)

Why do we need color specifically ?

Reply:
For corresponding visualization changes in clarity-tools (you'll see a PR there soon). Although, this is optional and only for visualization aid. The reason I grabbed it was because we already had access to computed styles because we have to check "visibility" property and it was a cheap lookup.

Reply:
What's the format of color?

Reply:
It's rgba. Based on feedback, I've put it behind a config so we don't have to worry about it in most cases but it does help at the time of a demo.